CHANGELOG.md

```diff
@@ -32,13 +25,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Internal
 
+# Past Releases
+
+## [2.3.0] - 2024-02-21
```

Comment:
I think technically the deployment target bump would necessitate a major version bump but I'm not too concerned about it.

Reply:
+1, I'd do a 3.0 here, since if someone is bound to only 2.0 in their profile and still using iOS 14, they'll get this update and then it'll error... I think

Reply:
that's a good point, here's the new PR: #482

Reply:
I don't think I would consider that a major breaking change, but I guess I don't feel too strongly about it. I don't want to get into a pattern of frequent major releases of Blueprint.